site/docs/3.5.3/404.html

```diff
@@ -15,36 +15,18 @@
         
 
 
-        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
-        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+        <link rel="stylesheet" href="css/bootstrap.min.css">
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&Courier+Prime:wght@400;700&display=swap" rel="stylesheet">
         <link href="css/custom.css" rel="stylesheet">
         <script src="js/vendor/modernizr-2.6.1-respond-1.1.0.min.js"></script>
 
         <link rel="stylesheet" href="css/pygments-default.css">
-        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
+        <link rel="stylesheet" href="css/docsearch.min.css" />
         <link rel="stylesheet" href="css/docsearch.css">
 
         
-        <!-- Matomo -->
```

Comment:
why delete the following?

Reply:
```
git cherry-pick 31fbf013e7cba1deeb359c0af17295488cf66461
git cherry-pick 3b892bd632502c56c1f76f858a4531f76118024b
git cherry-pick e395f33dd37f212b6e95b08467e95b58acd73d73
git cherry-pick 03f7200d5b6c63a732162b4762f72ac94a64a3f8
cd docs
SKIP_API=1 bundle exec jekyll build
```

Is this command correct? The code was all generated through commands, and I didn't manually edit any of it.

Reply:
The commands look correct to me

Reply:
1. Does it have to be generated on Linux, or can it also be done on macOS? I generated the code again on another Mac and it matches this PR.
2. Later, I'll ask @panbingkun  to give it a try as well and see if the same issue persists.

Reply:
I generated the doc on macOS too. If it is hard to fix the minor issues, we can still merge this to unblock the styling issues. Thanks for looking into this.

Reply:
I also had the same result locally, as follows:
<img width="1236" alt="image" src="https://github.com/user-attachments/assets/57fc54bc-6367-40ec-a27e-8541f544df6f" />

Reply:
Maybe you can try:
SKIP_API=1 PRODUCTION=1 bundle exec jekyll build

Reply:
<img width="1237" alt="image" src="https://github.com/user-attachments/assets/79452c44-3a0d-4ab1-bcd7-6ab15a4ca30f" />

Reply:
@panbingkun 
Could you help submit a new pr? I'm a bit busy today. Also, it looks like `configuration.html` needs to be edited manually.